qa/tasks/vstart_runner.py

```diff
@@ -1393,7 +1393,7 @@ def exec_test():
         log.info('\nrunning vstart.sh now...')
         # usually, i get vstart.sh running completely in less than 100
         # seconds.
-        remote.run(args=args, env=vstart_env, timeout=(3 * 60))
+        remote.run(args=args, env=vstart_env, timeout=(10 * 60))
```

Comment:
This change is not really required on squid since #56888 hasn't been backported. However, it's good to have to avoid "Ceph API test" failures.

Reply:
sounds good, i am merging this btw to unblock other PRs